README.md

```diff
@@ -1 +1,133 @@
-# CR-portfolio-project
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="media/png/screenshot.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>Portfolio</b></h3>
```

Comment:
[OPTIONAL] I think it would be better to use an image that represents your portfolio better than just the form contact since you want a complete website by the end of all milestones. For example, you can use a picture from your Intro section.